script/subtitles/imscsubtitles.js

```diff
@@ -138,10 +212,33 @@ define('bigscreenplayer/subtitles/imscsubtitles',
         }
       }
 
+      function modifyStyling (xml) {
+        if (liveSubtitles) {
+          xml.head.styling.initials = defaultStyleOpts.initials;
+        }
+        return xml;
+      }
+
+      function timeIsValid (time) {
```

Comment:
Is this another candidate for timeutils? Or is it IMSC subs specific?

Reply:
timeIsValid in this context is very live subtitle specific and is just our way of protecting us from requesting segments that are not needed when the player time isn't correct yet.

Reply:
👍